python/llm/example/NPU/HF-Transformers-AutoModels/Multimodal/minicpm_v_2_6.py

```diff
@@ -40,7 +40,7 @@
     parser.add_argument("--max-prompt-len", type=int, default=960)
     parser.add_argument("--disable-transpose-value-cache", action="store_true", default=False)
     parser.add_argument("--intra-pp", type=int, default=2)
-    parser.add_argument("--inter-pp", type=int, default=2)
+    parser.add_argument("--inter-pp", type=int, default=4)
```

Comment:
set `inter-pp` and `intra-pp` to `None`?

Reply:
Done